app/Http/Controllers/Api/ApiController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use League\Fractal\Manager;
+use League\Fractal\Resource\Item;
+use League\Fractal\Resource\Collection;
+use League\Fractal\Serializer\DataArraySerializer;
+use Gladiator\Http\Controllers\Controller as BaseController;
+
+class ApiController extends BaseController
+{
+    /**
+     * @var League\Fractal\Manager
+     */
+    protected $manager;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract
+     */
+    protected $transformer;
+
+    /**
+     * Format and return a collection response.
+     *
+     * @param  object  $data
+     * @param  int  $code
+     * @param  array  $meta
+     * @param  null|object  $transformer
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function collection($data, $code = 200, $meta = [], $transformer = null)
+    {
+        $collection = new Collection($data, $this->setTransformer($transformer));
+
+        return $this->transform($collection, $code, $meta);
+    }
+
+    /**
+     * Format and return a single item response.
+     *
+     * @param  object  $data
+     * @param  int  $code
+     * @param  array  $meta
+     * @param  null|object  $transformer
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function item($data, $code = 200, $meta = [], $transformer = null)
+    {
+        $item = new Item($data, $this->setTransformer($transformer));
+
+        return $this->transform($item, $code, $meta);
+    }
+
+    /**
+     * Manage and finalize the data transformation.
+     *
+     * @param  object  $data
+     * @param  int  $code
+     * @param  array  $meta
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function transform($data, $code = 200, $meta = [])
+    {
+        $data->setMeta($meta);
+
+        $this->manager = new Manager;
```

Comment:
Is the manager used outside this method? Might not need it to be a property on the class if not.